src/core/Akka.API.Tests/verify/CoreAPISpec.ApprovePersistenceSqlCommon.Net.verified.txt

```diff
@@ -272,7 +272,7 @@ namespace Akka.Persistence.Sql.Common.Journal
         public long Offset { get; }
         public Akka.Actor.IActorRef ReplyTo { get; }
     }
-    public abstract class SqlJournal : Akka.Persistence.Journal.AsyncWriteJournal, Akka.Actor.IActorStash, Akka.Actor.IWithUnboundedStash, Akka.Dispatch.IRequiresMessageQueue<Akka.Dispatch.IUnboundedDequeBasedMessageQueueSemantics>
+    public abstract class SqlJournal : Akka.Persistence.Journal.AsyncWriteJournal, Akka.Actor.IActorStash, Akka.Actor.IWithUnboundedStash, Akka.Actor.IWithUnrestrictedStash, Akka.Dispatch.IRequiresMessageQueue<Akka.Dispatch.IUnboundedDequeBasedMessageQueueSemantics>
```

Comment:
This makes me think that we might have to update all of the Akka.Persistence plugins, since the journal currently uses an unbounded stash. I don't think that should be important unless there's something on the normal `IWithUnrestrictedStash` interface that could cause a `MissingMethodException` but it looks to me like there isn't any. Just wanted to make note of this in case I'm wrong.

Reply:
Not really because the journal is still implementing `IWithUnboundedStash` which is more constrained. So, the mailbox is going to be selected based on it and not `IWithUnrestrictedStash`.